src/MdDocs.ApiReference/Model/_Helpers/CSharpDefinitionFormatter.cs

```diff
@@ -684,18 +685,18 @@ private static string GetLiteral(TypeReference typeReference, object value)
                 // on error (e.g. a value not defined in the enum), just return the plain value
                 else
                 {
-                    return value?.ToString() ?? "";
```

Comment:
We no longer need `?` here and below because we handle all nulls in the first case